components/drops/Drops.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div ref="container">
+    <div class="has-text-weight-bold is-size-5 mb-7">
```

Comment:
```suggestion
    <div class="title is-2 mb-7">
```

Reply:
done.
not sure why `is-2` looks too large to me, but i went with it

Reply:
`is-2` = 40px
<img width="304" alt="Capture d’écran 2023-06-22 à 3 10 24 PM" src="https://github.com/kodadot/nft-gallery/assets/9987732/94d91f88-f060-4b9e-9bbb-5b5e455ff9ed">